pandas/_libs/groupby.pyx

```diff
@@ -163,14 +164,21 @@ def group_cumprod_float64(float64_t[:, :] out,
                 if val == val:
                     accum[lab, j] *= val
                     out[i, j] = accum[lab, j]
+                if val != val:
```

Comment:
This can just be `else`?

Reply:
Yeah, that makes more sense. I changed that for both `cumsum` and `cumprod`.